### DISCUSSION TITLE
Mismatch The Comparison of The Files and The Folders

### DISCUSSION THREAD

**Discussion by alprozbk** [WinMerge/winmerge#2302] [Q&A] [answered]
Hello,
I have selected the folders in different drives to see file content differences between them. I realized the progress comparing by size.  I have the same folder names, the same size and the same date, but application brands them unique. The results were the same what method I chose for the comparison.

I have attached a screenshot and drew a red rectangle on some of the samples. But must of the other folders have this issue too. I am not sure Non-English characters in the names are causing this, but I don't want to believe this application has the lack of feature and not handle international characters. I really appreciate your help to know why this happens and how to solve.
  
![Screenshot_1](https://github.com/WinMerge/winmerge/assets/47886388/9ee628e9-c1a6-487b-b7ad-9d388f8fdaa9)

Regards,
Alper

**Comment by sdottaka** [comment, marked as answer]
I think this is the same issue as #2121.
Unfortunately, WinMerge is currently unable to ignore Unicode Normalization differences in filenames.

**Reply by alprozbk** [reply]
Hello,
Thanks for the info, and your answer looks like fair enough. 

But to walk around the issue for now, it would be better to find an bulk name conversion app that can change and match the UTF codes of the file and folder names at both sides. We can process the both side, then analyze by WinMerge. 

Do you know an freeware application or project to do so? This would help and solve the issue.

Regards,
Alper

**Reply by sdottaka** [reply]
Unfortunately, I don't know of any such application, but the URL below appears to do something similar with Powershell.
(I haven't tried it, but I think you may have to change `X:` or `FormC` depending on the situation.)

https://superuser.com/questions/1097217/determine-and-change-filename-encoding-on-windows/1111543#1111543
